ext/nnstreamer/tensor_source/tensor_src_tizensensor.c

```diff
@@ -907,27 +902,25 @@ gst_tensor_src_tizensensor_event (GstBaseSrc * src, GstEvent * event)
  * @brief Get possible GstCap from the configuration of self.
  */
 static GstCaps *
-_ts_get_gstcaps_from_conf (GstTensorSrcTIZENSENSOR *self)
+_ts_get_gstcaps_from_conf (GstTensorSrcTIZENSENSOR * self)
 {
   TizenSensorSpec *spec;
   gchar *tensor;
   GstCaps *retval;
-  gchar *typestr = NULL;
 
-  spec = g_hash_table_lookup (tizensensors, self->type);
+  spec = g_hash_table_lookup (tizensensors, (gpointer) self->type);
 
   if (FALSE == self->configured || SENSOR_ALL == self->type || NULL == spec) {
-    tensor = g_strdup_printf("other/tensor; other/tensors, num_tensors=1");
+    tensor = g_strdup_printf ("other/tensor; other/tensors, num_tensors=1");
   } else {
-    typestr = gst_tensor_get_type_string (spec->tinfo.type);
+    const gchar *typestr = gst_tensor_get_type_string (spec->tinfo.type);
     g_assert (typestr && typestr[0]);
-    tensor = g_strdup_printf("other/tensor, dimension=%u:%u:%u:%u, type=%s ; "
+    tensor = g_strdup_printf ("other/tensor, dimension=%u:%u:%u:%u, type=%s ; "
```

Comment:
TODO: consider to use gst_tensor_get_dimension_string()

Reply:
Fixed.